core/src/main/java/io/questdb/cairo/TableWriter.java

```diff
@@ -5580,6 +5349,230 @@ private void processPartitionRemoveCandidates0(int n) {
         }
     }
 
+    private long processWalBlock(
```

Comment:
this method is unchanged, it was needlessly "public". I made it "private" hence IDE refactoring.